app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt

```diff
@@ -59,23 +59,28 @@ class LoginActivityTest {
         Log.d(TAG, "running testUIViewsPresenceOnLoad..")
 
         // checks if email input field is present
+        onView(withId(R.id.email)).perform(scrollTo())
         onView(withId(R.id.email)).check(matches(isDisplayed()))
 
         // checks if password input field is present
+        onView(withId(R.id.password)).perform(scrollTo())
         onView(withId(R.id.password)).check(matches(isDisplayed()))
 
-        // checks if login button is present
+        //checks if login button is present
+        onView(withId(R.id.log_in)).perform(scrollTo())
         onView(withId(R.id.log_in)).check(matches(isDisplayed()))
 
         // checks if forgot password button is present
+        onView(withId(R.id.forgot_password)).perform(scrollTo())
         onView(withId(R.id.forgot_password)).check(matches(isDisplayed()))
 
         // checks if checkbox is present
+        onView(withId(R.id.customer_server)).perform(scrollTo())
         onView(withId(R.id.customer_server)).check(matches(isDisplayed()))
 
         // checks if sign up button is present
-        onView(withId(R.id.sign_up)).perform(scrollTo())
-        onView(withId(R.id.sign_up)).check(matches(isDisplayed()))
+        onView(withId(R.id.login_activity_sign_up)).perform(scrollTo())
+        onView(withId(R.id.login_activity_sign_up)).check(matches(isDisplayed()))
```

Comment:
Unrelated

Reply:
@iamareebjamal As the UI tests were failing that time and on going through the logs what I understood that time was that there was a trouble differentiating between the id "sign_up" which is present in both the Login and SignUp activity. Hence, I tried changing it for the Login Activity tests and it did pass the UI tests in Android studio. But, now I see that this doesn't turn up, there are some other issues. I will change this back to the original, as I have done in one of the other PRs.